activitylog/postgres/init.go

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Abstract Machines
+// SPDX-License-Identifier: Apache-2.0
+
+package postgres
+
+import (
+	_ "github.com/jackc/pgx/v5/stdlib" // required for SQL access
+	migrate "github.com/rubenv/sql-migrate"
+)
+
+func Migration() *migrate.MemoryMigrationSource {
+	return &migrate.MemoryMigrationSource{
+		Migrations: []*migrate.Migration{
+			{
+				Id: "activities_01",
+				Up: []string{
+					`CREATE TABLE IF NOT EXISTS activities (
+						id 			UUID NOT NULL DEFAULT gen_random_uuid() PRIMARY KEY,
+						operation	VARCHAR NOT NULL CHECK (operation <> ''),
```

Comment:
No need for this check, since we don't do that in the rest of the codebase. We will perform this check during validation.